Comments/src/handlers/HCommentBrowser.cpp

```diff
@@ -40,13 +40,13 @@ HCommentBrowser* HCommentBrowser::instance()
 void HCommentBrowser::keyPressEvent(Visualization::Item* item, QKeyEvent *event)
 {
 	if (event->modifiers() == Qt::ControlModifier && event->key() == Qt::Key_F11)
+	{
 		if (auto scene = item->scene())
 			for (auto view : scene->views())
 				if (dynamic_cast<Visualization::MainView*>(view))
 					view->centerOn(item);
-
-
-	// Accept all events and prevent them from being propagated to parents.
+	}
+	else GenericHandler::keyPressEvent(item, event);
```

Comment:
Hmm, the fact that I even put a comment in the original code, to note explicitly that I want to accept all events and not propagate them, makes me think that there was a good reason for that. Unfortunately, not having documented the reason, I can't remember what it is.

Let's leave your version for now, but you could please add a comment at the end of the method:
`// TODO: Is it OK to propagate events to parents or should we just accept all events? Propagating at least some events is necessary for updating an InfoNode`